src/lib/components/AimeVideoPlayer/AimeVideoPlayer.js

```diff
@@ -0,0 +1,96 @@
+import React, { PureComponent } from "react";
+import PropTypes from "prop-types";
+import ReactPlayer from "react-player";
+import {
+  componentPropTypes,
+  defaultComponentPropTypes,
+} from "../../utils/componentPropTypes";
+import styles from "./AimeVideoPlayer.module.scss";
+
+export default class AimeVideoPlayer extends PureComponent {
+  static propTypes = {
+    url: PropTypes.string.isRequired,
+    controlsVimeo: PropTypes.bool,
```

Comment:
I haven't had a closer look at the video player, I just wanted to suggest we keep all this configuration stuff to a minimum … so, unless we are sure we need it, I wouldn't even add all these props. Especially not the autoplay one, because I hope we will never do it!

I think it also makes things a little complicated to add configurations per player …? Happy to talk about this in a call!

Reply:
> I haven't had a closer look at the video player, I just wanted to suggest we keep all this configuration stuff to a minimum … so, unless we are sure we need it, I wouldn't even add all these props. Especially not the autoplay one, because I hope we will never do it!
> 
> I think it also makes things a little complicated to add configurations per player …? Happy to talk about this in a call!

Hey @rin ,

Yeah this is just getting to know the player right now and figuring out which props are essential, the auto play will be needed so we can have just one click of the play button.
Still looking at if that's the best solution.
Happy to jump on a call and bounce some ideas around 😃

Reply:
Hey @rin if you have any suggestions on what to remove I will happily look into it. right now it feels like what we have is what we need but that's just my perspective 😄